src/theme/defaultTheme/textArea.ts

```diff
@@ -0,0 +1,185 @@
+export const textArea = {
+  wrapper: "relative w-full",
+  suffixWrapper:
+    "flex items-center justify-end pb-2 absolute inset-y-0 right-0 bg-transparent z-10",
+  size: {
+    sm: {
+      base: {
+        default: "h-[73px] p-1 text-cxs rounded-lg",
```

Comment:
@crstnmac Can you check once if the padding for all different sizes is the same as the design file?

Reply:
fixed the padding and borderRadius according to the design